commcare_connect/opportunity/views.py

```diff
@@ -138,6 +138,10 @@ def form_valid(self, form):
         if end_date:
             opportunity.end_date = end_date
         response = super().form_valid(form)
+        if opportunity.auto_approve_visits:
```

Comment:
I dont think you want to backfill this automatically. I think its fine to only have it affect forms going forward, but otherwise, it could be an option. I think people might accidentally do something they don't intend to if it always happens like this.

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/294/commits/b1698ae22d0906989979e59cafd6dc51ab214630